src/ds3_library_exports.h

```diff
@@ -0,0 +1,19 @@
+//
```

Comment:
This should use the correct copyright details.

Reply:
Should have looked more closely at what CLion automatically created :p